DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditCentreWebsiteDetails.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model EditCentreWebsiteDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Edit Centre Website Details" : "Edit Centre Website Details";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names=
+                        "@(new[] { nameof(Model.CentreTelephone), nameof(Model.CentreEmail), nameof(Model.CentrePostcode), nameof(Model.OpeningHours), nameof(Model.CentreWebAddress), nameof(Model.OrganisationsCovered), nameof(Model.TrainingVenues), nameof(Model.OtherInformation) })"/>
+    }
+
+    <h1 id="page-heading" class="nhsuk-heading-xl">Edit centre website details</h1>

```

Comment:
Could we keep the title consistent with the wireframe and previous page "centre content on DLS website" rather than "website details"?